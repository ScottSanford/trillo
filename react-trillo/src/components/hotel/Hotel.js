import React from 'react'
import './Hotel.css'
import Gallery from '../gallery/Gallery'
import Details from '../details/Details'
import CallToAction from '../call-to-action/Call-to-action'
import Overview from '../overview/Overview'

export default function Hotel({ details }) {

  const { hotelImages } = details

  return (
    <main className="hotel-view">
      <Gallery images={hotelImages} />
      <Overview hotel={details} />
      <Details />
      <CallToAction />
    </main>
  )
}