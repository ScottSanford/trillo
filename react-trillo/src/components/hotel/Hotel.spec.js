import React from 'react'
import { shallow } from 'enzyme'
import Hotel from './Hotel'

describe('Hotel Component', () => {

	const minProps = {
		details: {}
	} 

	it('renders without exploding', () => {
		const wrapper = shallow(<Hotel {...minProps} />)
		expect(wrapper.length).toEqual(1)
	})

	it('should match snapshot', () => {
		const wrapper = shallow(<Hotel {...minProps} />)
		expect(wrapper).toMatchSnapshot()
	})

	
})