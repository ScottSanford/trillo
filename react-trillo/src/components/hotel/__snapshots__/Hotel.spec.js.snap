// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hotel Component should match snapshot 1`] = `
<main
  className="hotel-view"
>
  <Gallery />
  <Overview
    hotel={Object {}}
  />
  <div
    className="detail"
  >
    <Description />
    <UserReviews />
  </div>
  <CallToAction />
</main>
`;
